x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/logical/show/ShowFunctions.java

```diff
@@ -63,6 +63,7 @@ public List<List<Object>> values(FunctionRegistry functionRegistry) {
             row.add(signature.description());
             row.add(collect(signature, EsqlFunctionRegistry.ArgSignature::optional));
             row.add(signature.variadic());
+            row.add(signature.isAggregation());
```

Comment:
@luigidellaquila is this one ok with you?

Reply:
Yes, absolutely fine